plugins/modules/net_tools/nmcli.py

```diff
@@ -20,10 +20,10 @@
 - nmcli
 description:
     - Manage the network devices. Create, modify and manage various connection and device type e.g., ethernet, teams, bonds, vlans etc.
-    - 'On CentOS 8 and Fedora >=29 like systems, the requirements can be met by installing the following packages: NetworkManager-libnm,
-      libsemanage-python, policycoreutils-python.'
-    - 'On CentOS 7 and Fedora <=28 like systems, the requirements can be met by installing the following packages: NetworkManager-glib,
-      libnm-qt-devel.x86_64, nm-connection-editor.x86_64, libsemanage-python, policycoreutils-python.'
+    - 'On CentOS/RHEL 8 and Fedora systems, the requirements can be met by installing the following packages: NetworkManager-libnm,
```

Comment:
Why did you remove the information on Fedora versions?

Reply:
All versions with python2 as default are not supported anymore.

Reply:
So what? Why remove information that is potentially valuable just because something is not supported anymore?

Reply:
Please keep older information as well as people might require it.